Havit.Blazor.Components.Web.Bootstrap/Navigation/HxSidebar.razor

```diff
@@ -21,8 +21,16 @@
 					</CascadingValue>
 				</HxNav>
 				<div class="hx-sidebar-toggler d-md-flex" role="button" @onclick="HandleCollapseToggleClick"
-					 @onclick:stopPropagation="true">
-					<HxIcon Icon="@(Collapsed ? ExpandIcon : CollapseIcon)" />
+					@onclick:stopPropagation="true">
+                    
+					@if (TogglerTemplate is null)
+					{
+						<HxIcon Icon="@(Collapsed ? ExpandIcon : CollapseIcon)" />
+					}
+					else
+					{
+						@TogglerTemplate
```

Comment:
The template should receive a context to be able to render different content for Collapsed/Expanded state (e.g. `SidebarTogglerTemplateContext` or `SidebarTogglerState` with `bool Collapsed` property)

```csharp
[Parameter] public RenderFragment<SidebarTogglerTemplateContext> TogglerTemplate { get; set; }
```